docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.9"
+services:
+  app:
+    image: golang
```

Comment:
One final small change here. Lets version lock this version of Go to what we've been using for testing in our Deputy container to ensure that we're working from the same baseline here

`golang:1.13.5`

Reply:
This change is now implemented.